pkg/pillar/cmd/domainmgr/metric.go

```diff
@@ -17,9 +17,10 @@ func metricsTimerTask(ctx *domainContext, hyper hypervisor.Hypervisor) {
 	log.Functionln("starting metrics timer task")
 	getAndPublishMetrics(ctx, hyper)
 
-	// Publish 4X more often than zedagent publishes to controller
+	// Publish 20X more often than zedagent publishes to controller
+	// to reduce effect of quantization errors
 	interval := time.Duration(ctx.metricInterval) * time.Second
-	max := float64(interval) / 4
+	max := float64(interval) / 20
```

Comment:
NIT: should this be a constant since it is used at more then one place?

Reply:
It is so ad-hoc it doesn't make much sense to define it just because we use it twice.